site/main.css

```diff
@@ -85,33 +186,52 @@ hr.footnotes { width: 40%; }
 a.footnote
 {
     text-decoration: none;
+    margin-right: 0.4rem;
 }
 
 ul.toc { list-style: none; }
 
-h5 { font-size: 100%; }
+h5 { font-size: 1.6rem; }
+
+code, kbd, samp
+{
+    color: #1a8a66;
```

Comment:
My feeling regarding fallback properties and fallback values in `var()` statements is that we should either use CSS variables and keep them correctly defined, or not use them at all.

Since any decent browser released after 2016 supports them, and it's quite hard to imagine a person interested in a new SSG still using a five year old browser, I would be inclined to go all-in with variables.

Reply:
My reasoning for this:without a tool like PostCSS + preset-env doing this automatically, I've done it manually. Major browser support of `var()` is good, but minor browsers ... not so much. I was not sure if you wanted to support projects like [Netsurf](http://www.netsurf-browser.org/documentation/progress.html#CSSFeatures). If the goal is maintainability, then yes, I would agree with axing it. If the goal is to support old or indie browsers, it makes sense to keep and take on that burden. Removing is :ok:

Reply:
Netsurf support is an interesting point. Philosophically, of course I'd like to support Netsurf. Pragmatically, it's better to put effort into improving Netsurf itself, since it needs modern CSS features to remain viable.